include/roboteam_utils/World.hpp

```diff
@@ -28,7 +26,6 @@ typedef struct WorldStates {
     // World futureWorld;       // TODO: Implement this
 
     rtt::Field field;           // The field of the world
-    proto::SSL_Referee referee; // The latest referee command
```

Comment:
How do you plan on sending the referee data? Do you intend to also make a new data class for that?

Reply:
I was thinking about keeping this as a proto object because of two reasons.
First, this referee object has an insane amount of booleans and enums, and it would take quite a while to write a class that resembles this. Second of all, the referee is practically unrelated to world, so I was thinking of still including it in the stuff that is sent to AI, but not in the structs that resemble the world. (This can be changed later)